masterlist.yaml

```diff
@@ -112,9 +238,69 @@ globals:
         text: '[최신 LOOT 스레드](%1%).'
       - lang: zh_CN
         text: '[最新的LOOT版本](%1%)。'
+      - lang: da
+        text: '[Seneste LOOT-tråd](%1%).'
       - lang: ja
         text: '[LOOTの最新スレッドを開く](%1%)（英語）'
     subs: [ 'https://loot.github.io/latest-thread/' ]
+# SKYRIM SE Latest
+  - type: say
+    condition: 'file("../SkyrimSE.exe") and version("../SkyrimSE.exe", "1.0.0.0", <)'
+    content:
+      - lang: en
+        text: 'It appears you do not have the latest version of Skyrim Special Edition.'
+      - lang: da
+        text: 'Det ser ikke ud til at du har den seneste version af Skyrim Special Edition.'
+      - lang: ja
+        text: 'Skyrim Special Editionが最新版ではないようです。'
+# SKSE Latest
+  - type: say
+    condition: 'file("../skse64_loader.exe") and version("../skse64_loader.exe", "0.2.0.6", <)'
+    content:
+      - lang: en
+        text: 'It appears you do not have the [latest version of SKSE64](http://skse.silverlock.org). Some mods may require functionality added in the latest version of SKSE64 to work.'
+      - lang: ru
+        text: 'Возможно у вас не установлена [новейшая версия SKSE64](http://skse.silverlock.org). Для работы некоторых модов может требоваться функционал, добавленный в последней версии SKSE64'
+      - lang: es
+        text: 'Parece que no tienes la última versión de SKSE64. la nueva versión se puede encontrar [aquí](http://skse.silverlock.org/).'
+      - lang: ko
+        text: '[SKSE64의 최신 버전](http://skse.silverlock.org)이 설치 되지 않은 것같습니다. 일부 모드가 동작하려면 SKSE64의 최신 버전에서 추가된 기능을 필요로 할 수도 있습니다.'
+      - lang: zh_CN
+        text: '你的[SKSE64](http://skse.silverlock.org)似乎不是最新的。有些插件可能需要最新的SKSE才能工作。'
+      - lang: ja
+        text: '[SKSE64](http://skse.silverlock.org)が最新版ではないようです。一部のModを動作させるには最新版のSKSEで追加された機能が必要となる場合があります。'
+  - type: say
+    condition: 'not version("../skse64_loader.exe", "0.2.0.6", <)'
+    content:
+      - lang: en
+        text: 'Your SKSE64 is up-to-date.'
+      - lang: ru
+        text: 'Ваш SKSE64 обновлен.'
+      - lang: es
+        text: 'Tienes la última versión de SKSE64'
+      - lang: ko
+        text: 'SKSE64가 최신버전입니다.'
+      - lang: zh_CN
+        text: '你的SKSE64是最新的。'
+      - lang: da
+        text: 'Din SKSE64 er opdateret.'
+      - lang: ja
+        text: 'SKSE64は最新版です。'
+  - type: error
+    condition: 'version("../skse64_loader.exe", "0.2.0.6", ==) and version("../SkyrimSE.exe", "1.0.0.0", !=)'
+    content:
+      - lang: en
+        text: 'Your version of skse64 is not compatible with your version of Skyrim Special Edition.'
+      - lang: da
+        text: 'Din version af skse64 er ikke kompatibel med din version af Skyrim Special Edition.'
+      - lang: ja
+        text: 'このバージョンのskse64は使用中のSkyrim Special Edition との互換性がありません。'
+# SKSE64 plugins requirement
+  - <<: *requireSKSE64
+    subs: [ 'Achievements Mods Enabler' ]
+    # url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/245' ]
```

Comment:
A more useful way of encoding the link would be to include it in the message substitution, e.g. `subs: [ '[Achievements Mods Enabler](https://www.nexusmods.com/skyrimspecialedition/mods/245)' ]`. That way it'll show up as a hyperlink in the UI and won't look slightly confusing - initially I wondered why you'd left invalid syntax commented out.

Reply:
Can't claim credit for the code, it was copied from FO4 masterlist. Thanks for the tip.